src/components/Box/index.tsx

```diff
@@ -28,7 +28,8 @@ export type BoxProps = BackgroundSetProps &
   FlexChildSetProps &
   GridChildSetProps &
   LayoutSetProps &
-  TransformSetProps;
+  TransformSetProps &
+  PolymorphProps;
```

Comment:
This change alone is fine, but the commit title doesn't really express the fact that it's a functional change. Maybe something like `Allow overriding Box component's rendered element`?

Reply:
That works